devdox/app/routes/api_keys_route.py

```
@@ -0,0 +1,36 @@
+from fastapi import APIRouter, Depends
+from starlette import status
+from starlette.responses import JSONResponse
+
+from app.services.api_keys_service import PostApiKeyService
+from app.utils import constants
+from app.utils.api_response import APIResponse
+from app.utils.auth import get_authenticated_user, UserClaims
+
+router = APIRouter()
+
+@router.post(
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)